src/continuum_opacity/opacity_H.jl

```diff
@@ -44,29 +44,56 @@ function _ndens_Hminus(nH_I_div_partition, ne, T, ion_energy = _H⁻_ion_energy)
 end
 
 
+const _Hminus_bf_table = SA[1250.0 5.431;   1500.0 6.512;   1750.0 7.918;   2000.0 9.453;
+                            2250.0 11.08;   2500.0 12.75;   2750.0 14.46;   3000.0 16.19;
+                            3250.0 17.92;   3500.0 19.65;   3750.0 21.35;   4000.0 23.02;
+                            4250.0 24.65;   4500.0 26.24;   4750.0 27.77;   5000.0 29.23;
+                            5250.0 30.62;   5500.0 31.94;   5750.0 33.17;   6000.0 34.32;
+                            6250.0 35.37;   6500.0 36.32;   6750.0 37.17;   7000.0 37.91;
+                            7250.0 38.54;   7500.0 39.07;   7750.0 39.48;   8000.0 39.77;
+                            8250.0 39.95;   8500.0 40.01;   8750.0 39.95;   9000.0 39.77;
+                            9250.0 39.48;   9500.0 39.06;   9750.0 38.53;   10000.0 37.89;
+                            10250.0 37.13;  10500.0 36.25;  10750.0 35.28;  11000.0 34.19;
+                            11250.0 33.01;  11500.0 31.72;  11750.0 30.34;  12000.0 28.87;
+                            12250.0 27.33;  12500.0 25.71;  12750.0 24.02;  13000.0 22.26;
+                            13250.0 20.46;  13500.0 18.62;  13750.0 16.74;  14000.0 14.85;
+                            14250.0 12.95;  14500.0 11.07;  14750.0 9.211;  15000.0 7.407;
+                            15250.0 5.677;  15500.0 4.052;  15750.0 2.575;  16000.0 1.302;
+                            16100.0 0.8697; 16200.0 0.4974; 16300.0 0.1989]
+
+const _Hminus_bf_cross_section_interp = LinearInterpolation(view(_Hminus_bf_table, :, 1),
+                                                            view(_Hminus_bf_table, :, 2),
+                                                            extrapolation_bc=Throw())
+
 """
-    _Hminus_bf_cross_section(ν)
+    _Hminus_bf_cross_section(λ, ion_energy_H⁻)
 
-Compute the H⁻ bound-free cross-section, which has units of cm^2 per H⁻ particle.
+Compute the H⁻ bound-free cross-section at a given wavelength (specified in Å). The cross-section 
+has units of megabarns per H⁻ particle and does NOT include a correction for stimulated emission.
 
-The cross-section does not include a correction for stimulated emission.
+This function linearly interpolates the table provided in Wishart (1979).
 """
-function _Hminus_bf_cross_section(ν::Real)
-    λ = c_cgs*1e8/ν # in Angstroms
-    # we need to somehow factor out this bounds checking
-    if !(2250 <= λ <= 15000.0)
-        throw(DomainError(λ, "The wavelength must lie in the interval [2250 Å, 15000 Å]"))
+function _Hminus_bf_cross_section(λ, ion_energy_H⁻)
+    Å_per_eV = 1e8 * (hplanck_eV * c_cgs)
+    max_λ_ionize = Å_per_eV/ion_energy_H⁻
+
+    last_table_λ = _Hminus_bf_table[size(_Hminus_bf_table, 1), 1]
+    last_table_cross_section = _Hminus_bf_table[size(_Hminus_bf_table, 1), 2]
+
+    @assert max_λ_ionize > last_table_λ
+
+    if λ < _Hminus_bf_table[1,1]
+        throw(DomainError(λ, "$(_Hminus_bf_table[1,1]) Å must not exceed the wavelength"))
```

Comment:
Is this `>` the right way around?

Reply:
Or the word "exceed" wrong?

Reply:
This was technically correct as written, but it definitely looked weird (I basically flipped the inequality). I changed it so that it makes more sense.

Reply:
Ooooooh, this took me way longer than it should have.